client/performance-profiler/components/dashboard-content/index.tsx

```diff
@@ -115,7 +115,7 @@ export const PerformanceProfilerDashboardContent = ( {
 			</div>
 
 			<Disclaimer />
-			{ displayMigrationBanner && (
+			{ displayMigrationBanner && ! is_wpcom && (
```

Comment:
I would instead change the value of `displayMigrationBanner` to use `is_wpcom` as parameter in https://github.com/Automattic/wp-calypso/blob/09a5033eb710c86b668946046ad92dcb27fb2675/client/performance-profiler/pages/dashboard/index.tsx#L144

where it seems it's getting the default `true`.

Reply:
That makes sense, and we will avoid adding the extra condition there. I am doing that 👍

Reply:
Done in [afdb0fe](https://github.com/Automattic/wp-calypso/pull/94892/commits/afdb0fe0a5039299dd221104322759c9ce64c726)